perf/bonnie.py

```diff
@@ -75,6 +89,14 @@ def test(self):
         if process.system(cmd, shell=True, ignore_status=True):
             self.fail("test failed")
 
+    def tearDown(self):
+
+        '''
+        Cleanup of disk used to perform this test
+        '''
+        self.log.info("Unmounting directory %s" % self.scratch_dir)
```

Comment:
Syntax `self.log.info("Unmounting directory %s", self.scratch_dir)` is preferred:
https://docs.python.org/2/library/logging.html#logging.Logger.debug